src/ec/curve25519/curve25519.rs

```diff
@@ -12,7 +12,9 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-//! Elliptic curve operations on Curve25519.
+//! Elliptic curve operations and schemes using Curve25519.
```

Comment:
This was an attempt to distinguish between the sorts of things in `ops` (elliptic curve arithmetic operations) and `ed25519` (a signature scheme).